clients/tests/CMakeLists.txt

```diff
@@ -415,9 +415,48 @@ if (NOT WIN32)
 endif()
 
 if (NOT WIN32)
-  if (BUILD_WITH_ROCTX)
-    target_link_libraries(rocsparse-test PRIVATE -lroctx64)
+  set(ROCTX_PATH "" CACHE STRING "Path to the roctx library (directory containing libroctx64.so)")
+
+  find_path(ROCTRACER_INCLUDE_DIR
+    NAMES roctracer/roctx.h
+    HINTS
+    ${ROCTX_PATH}
+    ${ROCTX_PATH}/include
+    /opt/rocm/include
+    ${ROCTX_PATH}/../include
+    DOC "Path to the roctracer include directory containing roctx.h")
+
+  find_library(ROCTX_LIBRARY
+    NAMES roctx64
+    HINTS
+    ${ROCTX_PATH} # User-provided path
+    ${ROCTX_PATH}/lib
```

Comment:
On Debian, this path will be `<...>/${multi-arch}/lib` and on Fedora it will be `<...>/lib64`. It's probably not sufficient to check just lib.